syncer/db_test.go

```diff
@@ -151,12 +152,12 @@ func (s *testSyncerSuite) TestExecuteSQLSWithIgnore(c *C) {
 	c.Assert(err, IsNil)
 	dbConn, err := db.Conn(context.Background())
 	c.Assert(err, IsNil)
-	conn := &DBConn{
-		baseConn: &conn.BaseConn{
+	conn := &dbconn.DBConn{
```

Comment:
Should we move these tests into `dbconn` package?

Reply:
the logic of `syncer` package is very complex 😂 `ignoreDDLError` belongs to `syncer` now so `syncer/onlineddl` can't access it. We might refactor it in PRs like https://github.com/pingcap/dm/pull/1824